.github/PULL_REQUEST_TEMPLATE/feature.md

```diff
@@ -0,0 +1,18 @@
+#### [LEVEL]/[COMPONENT]
+| TASK | <!--- [jira TASK ID](jiraURL) -->             |
+| ---- | --------------------------------------------- |
+| DEMO | <!--- [[LEVEL]/[COMPONENT]](zeit now URL) --> |
```

Comment:
not sure if we really need this as now is going to add a message with the link as soon as it's deployed

Reply:
I supose that if we specify in the title the demo, maybe this direct link is not needed as you suggest